### DISCUSSION TITLE
Problems loading ISR pages with locale i18 enabled

### DISCUSSION THREAD

**Discussion by fxmb** [vercel/community#5421] [Help]
### Summary

When I add a locale like below I cannot load (or extremely slow / buggy) ISR generated pages. They timeout after some time, a few work and most of them just crash.

This is my setup:

_next-i18next.config.js_

```
/** @type {import('next-i18next').UserConfig} */
module.exports = {
  i18n: {
    defaultLocale: "de",
    locales: ["de"],

  },
}
```

I import this to my next.config as 

```
const { i18n } = require("./next-i18next.config")

module.exports = withStoreConfig({
  i18n,
  ....
```

Then I have routes on /[handle].tsx that **do not even use the locale (meaning I do not preload them using getServerSideProps())**. Locally it runs just fine but in preview / production the functions are timing out. I have the very strong suspicion it is because there is a /de/ added to the _next cache path. 

E.g.

`/_next/data/SlC_cCIi80TZYVas9qntH/[my-handle]`

has now become

`/_next/data/SlC_cCIi80TZYVas9qntH/de/[my-handle]`

Note how the /de/ is in there now. 

To me it is not clear what else could be causing this issue. 

Once I deactivate the locale it works just fine again. 

Is there any explanation for this? 

### Steps to Reproduce

See problem description

**Comment by AndryOre** [comment]
Same issue
